demos/http/http_demo_s3_upload/CMakeLists.txt

```diff
@@ -0,0 +1,44 @@
+set( DEMO_NAME "http_demo_s3_upload" )
+
+# Include HTTP library's source and header path variables.
+include( ${CMAKE_SOURCE_DIR}/libraries/standard/coreHTTP/httpFilePaths.cmake )
+
+# Demo target.
+add_executable(
+    ${DEMO_NAME}
+        "${DEMO_NAME}.c"
+        "../common/src/http_demo_utils.c"
+        ${HTTP_SOURCES}
+        ${HTTP_THIRD_PARTY_SOURCES}
+)
+
```

Comment:
I think it would be helpful to download the Baltimore Cybertrust root CA similar to how we do it for the Amazon Root CA.

Reply:
Yes, good point. Updated the CMake to download the Baltimore root CA alongside Amazon CA1.